docs/category/category.yaml

```
@@ -179,6 +179,42 @@ paths:
                 status: 401
                 code: UNAUTHORIZED
                 message: The requested URL requires user authorization.
+  /categories/subjects/names:
+    get:
+      security:
+        - cookieAuth: []
+      tags:
+        - Categories
+      summary: Finds and returns all subjects
```

Just a little suggestion to make summary more detailed
```suggestion
      summary: Finds and returns all subjects's names
```